### PR TITLE
[Feature]: The blobstore-cli supports setting background task switch #1898

### LINKED ISSUE #1898: [Feature]: The blobstore-cli supports setting background task switch
**Describe the solution you'd like**
- The blobstore-cli tool supports querying switch status
- The blobstore-cli tool supports enabling or disabling the background task switch

**Additional context**
The background tasks of blobstore include `disk_repair`, `balance`, `disk_drop`, `volume_inspect`, `shard_repair`, and `blob_delete`. The task switches are stored in Clustermgr. You can set the switch by requesting the "/config/set" interface of Clustermgr, and query through "/config/get" Switch status, for more convenient setting or query, we need to integrate it into the cli tool.

### PR DESCRIPTION
**Pull request by cloorc**
**What this PR does / why we need it**:

Support new feature requested in issue #1898 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #1898 

**Special notes for your reviewer**:

**Release note**:

```release-note
- `blobstore-cli` now support `enable` or `disable` background task switches
```

### CODE REVIEW

File: blobstore/cli/clustermgr/config_background.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	"github.com/cubefs/cubefs/blobstore/cli/common"
+	"github.com/cubefs/cubefs/blobstore/cli/common/flags"
+	"github.com/cubefs/cubefs/blobstore/cli/common/fmt"
+	"github.com/cubefs/cubefs/blobstore/common/rpc"
+)
+
+const notSet = "<not set>"
+
+var (
+	BackgroundTaskTypes      = []string{"disk_repair", "balance", "disk_drop", "volume_inspect", "shard_repair", "blob_delete"}
```
Comment (pengtianyue025) [inline blobstore/cli/clustermgr/config_background.go:33]: the task type already defined in `blobstore/common/proto/scheduler.go`,  you can use it

Comment (cloorc) [reply blobstore/cli/clustermgr/config_background.go]: Solved. Please have a look at https://github.com/cubefs/cubefs/pull/1997/commits/dca26cb1ef7ad422c4efc768ddd17442c72cc68b .

File: blobstore/cli/clustermgr/config_background.go [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+	showConfig(key, oldV, true)
+
+	if common.Confirm(fmt.Sprintf(
+		"To %s background task `%s` from `%s` --> `%s` ?", act, common.Loaded.Sprint(key),
+		common.Danger.Sprint(oldV), common.Normal.Sprint(value))) {
+		err := cli.SetConfig(ctx, &clustermgr.ConfigSetArgs{
+			Key:   key,
+			Value: value,
+		})
+		if nil != err {
```
Comment (pengtianyue025) [inline blobstore/cli/clustermgr/config_background.go:144]: The usual usage is `if err!=nil`

Comment (cloorc) [reply blobstore/cli/clustermgr/config_background.go]: Sorry, it's resulted from my c++ experience. Solved already. Please check the latest commit.

**Review by pengtianyue025 — APPROVED** [review]

**Comment by pengtianyue025** [comment]
LGTM

**Pull request merged**